components/list/list-item-mixin.js

```diff
@@ -512,6 +522,9 @@ export const ListItemMixin = superclass => class extends composeMixins(
 	}
 
 	resizedCallback(width) {
+		if (width < SLIM_COLOR_BREAKPOINT) this._slimColor = true;
+		else this._slimColor = false;
```

Comment:
This does tap into the existing resize observer on the list item, which I know we want to change. I have a story for doing that work and I have time to do it October 2, so I'm wondering if it's okay to get this piece in now for color indicator and move this around later as we move the resize observer, or if we want to wait on this until that all is done?

Reply:
These two lines could also be reduced to:
```javascript
this._slimColor = (width < SLIM_COLOR_BREAKPOINT);
```